examples/tilted_bottom_boundary_layer.jl

```diff
@@ -95,10 +95,19 @@ coriolis = ConstantCartesianCoriolis(f = 1e-4, rotation_axis = ĝ)
 # _perturbations_ away from the constant density stratification by imposing
 # a constant stratification as a `BackgroundField`,
 
-B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = 1e-5))
+N² = 1e-5 # s⁻¹ # background stratification
+B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = N²))
 
 # where ``N^2 = 10^{-5} \rm{s}^{-2}`` is the background buoyancy gradient.
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/CliMA/Oceananigans.jl/pull/3581/commits/727601d7c984d2eaf16031c2226c56cff37ee235